manifests/prometheus/dashboards.d/cell-memory.json

```diff
@@ -0,0 +1,550 @@
+{
+  "dashboard": {
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": "-- Grafana --",
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "editable": true,
+    "gnetId": null,
+    "graphTooltip": 1,
+    "id": null,
+    "iteration": 1666363845398,
+    "links": [],
+    "panels": [
+      {
+        "cards": {
+          "cardPadding": null,
+          "cardRound": null
+        },
+        "color": {
+          "cardColor": "#b4ff00",
+          "colorScale": "sqrt",
+          "colorScheme": "interpolatePlasma",
+          "exponent": 0.5,
+          "mode": "spectrum"
+        },
+        "dataFormat": "timeseries",
+        "datasource": null,
+        "description": "Amount of available memory advertised by rep over the actual amount of available memory on cell. >100% is in-debt as a result of overcommitting.",
```

Comment:
Perhaps

> The amount of uncommitted memory (reported by rep) as a percentage of the system's actual free memory
> (e.g. 18gb uncommitted/12gb free = 150%)
> A figure between x% and y% is healthy. Below that, we're underutilising. Beyond that, we risk exhausting memory in the event of an AZ failure

Reply:
That's not what it is - below, we're underutilizing. Above, we're risking apps crashing with OOM if they try to use all their memory.

Reply:
I think I'm missing the difference between the two statements. Is it that @AP-Hunt's suggestion makes sense as long as the last sentence is changed to mention crashing with OOM instead of AZ failures?

Reply:
Yes.

Though having said that, the latest version of these dashboards has been changed a bit and I'll have to adapt this sentence a bit more. Haven't added the latest changes to this PR because it requires some manual copypasta which i haven't done yet.